lib/elixir/lib/enum.ex

```diff
@@ -1879,11 +1876,60 @@ defmodule Enum do
       iex> Enum.min_max_by([], &String.length/1, fn -> {nil, nil} end)
       {nil, nil}
 
+  The fact this function uses Erlang's term ordering means that the
```

Comment:
```suggestion
  The fact that this function uses Erlang's term ordering means that the
```

Reply:
Good suggestion, @josevalim should I change both of these (as I copied it from min_by) in this PR?

Reply:
We can do them separately. I will also take the opportunity to run a grammar check on the whole doc.